### PR TITLE
Add Jonan to participants.

### PR DESCRIPTION
**Pull request by thejonanshow**
I would like to register for JS CraftCamp.

- [x] My pull request contains a JSON file `$name_or_nickname.json`
- [x] The JSON file follows the template at [https://github.com/jscraftcamp/website/blob/main/participants/\_template.json](https://jscraftcamp.org/register/) and contains the mandatory fields like `name`, `when`, `whatIsMyConnectionToJavascript` and `whatCanIContribute`
- [x] If I can NOT attend, I will either send another pull request removing my JSON file or an e-mail with the subject 'UNREGISTER' to team@jscraftcamp.org
- [x] I agree that the data I enter in the registration can be used for running the event, e.g. make a name tag, count me as participant. I acknowledge that my data are public on GitHub and I agree that I will be listed on the JSCraftCamp website as a participant.
- [x] I agree that photos and videos might be taken and published (e.g. on social media) during the event.
- [x] I understand that I might NOT get a T-Shirt, because they might be in production already
- [x] I asked my company about sponsoring (see open items at: https://github.com/orgs/jscraftcamp/projects/5/views/1)

Are you from a sponsoring company?

- [ ] Yes, I am from company ?????

### CODE REVIEW

**Comment by Narigo** [comment]
Hi @thejonanshow - thanks for your interest in the JSCraftCamp!

We closed the registration on Monday since we're already over the number of participants we can handle. That's at 100 participants per day. Lunch and space is planned for that amount of people. We were only merging people from sponsors after that. Thanks to some people unregistering again, we **do** have some spots left for Saturday. Would you be okay with only attending on Saturday? If you change your PR accordingly, we can merge it and make it possible for you to attend on Saturday.

Please join [the JSCC Discord](https://discord.gg/3mHhaXVEjP) if you have any questions about this! Joining the Discord is also a good way to connect with the broader community.

File: participants/jonan.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Jonan Scheffler",
+    "company": "DXOS",
+    "when": {
+        "friday": true,
```
Comment (Narigo) [inline participants/jonan.json:5]: Friday spots are all taken! If you want to participate on Saturday, please change this to `false` and keep `saturday` set to `true`.

```suggestion
        "friday": false,
```

Once you change this, I can merge the PR and you're admitted for tomorrow.

File: participants/jonan.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        "saturday": true
+    },
+    "iCanTakeNotesDuringSessions": true,
+    "tags": ["JavaScript", "TypeScript", "React"],
+    "whatIsMyConnectionToJavascript": "Relearning modern JavaScript after a few years away.",
+    "whatCanIContribute": "Career coaching, discussion mediation, karaoke thought leadership.",
+    "tShirt": {
+        "size": "3X",
```
Comment (Narigo) [inline participants/jonan.json:13]: Did you mean to set it to `3XL` here?
```suggestion
        "size": "3XL",
```

**Comment by thejonanshow** [comment]
Thank you for trying to accommodate me. I think I'm going to have to skip this one. I hope you have a lovely conference!

**Pull request closed without merging**